test_cases/widget_filter_by_connected_record.js

```diff
@@ -6,21 +6,30 @@ export default (folderName, Common) => {
             .click();
       });
 
-      it("Filtering ByDefault", () => {
+      it("Insert Data for Tests", () => {
```

Comment:
Shouldn't data-insert type actions be inside a before-each? or a before?

Reply:
Sure...I will put them there...they had been in the actual tests that they were used in before so that is even better.